src/lib_ccaligner/generate_approx_timestamp.cpp

```diff
@@ -19,6 +19,11 @@ void CurrentSub::printToSRT(const std::string& fileName, outputOptions printOpti
 {
     std::ofstream out;
     out.open(fileName, std::ofstream::app);
+    
+    if(!out.good())
+    {
+        LOG("Couldn't open file" + fileName);
```

Comment:
This is not a valid way to use it (hence the build failure). See the function and macro definition for more.